src/main/docker/gitlab-gitlabci.yml

```
@@ -0,0 +1,52 @@
+version: "3.6"
+services:
+    gitlab:
+        build: gitlab
+        container_name: gitlab
+        volumes:
+            - artemis-gitlab-data:/var/opt/gitlab
+            - artemis-gitlab-logs:/var/log/gitlab
+            - artemis-gitlab-config:/etc/gitlab
+        environment:
+            GITLAB_OMNIBUS_CONFIG: |
+                external_url "http://${GIT_SERVER_NAME}"
```

```suggestion
                external_url "${GIT_SERVER_NAME}"
```

Simply add the protocol to the environment variable as well :)